### ISSUE TITLE
Name of ‘Analyze equation' pane is not defined

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1126]
**Repro Steps:** 
1. Launch 'Calculator' and turn on narrator. 
2. Home screen will be displayed.​
3. Navigate through Menu using keyboard and select 'Graphing'. 'Graphing' Calculator opens.​
4. Navigate to equation edit filed and pressed context key.
5. Navigate to "Analyze equation" button and activate it.
6. “Key graph feature” pop up will open. 
7. Observe the issue.

**Actual Result:**
Name of ‘Analyze equation' pane is not defined. Narrator does read the name of “Key graph feature” pane when it opens after activating “Analyze equation” button.

**Expected Result:**
Name of ‘Analyze equation' pane should be defined. Narrator should read the name of “Key graph feature” pane when it opens after activating “Analyze equation” button.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.
